torch/onnx/utils.py

```diff
@@ -1370,7 +1370,7 @@ def _export(
         # We do not want to fail because we should still allow users to create
         # custom symbolic functions for opset>17
         warnings.warn(
-            f"Exporting to ONNX opset version {opset_version} is not supported. "
+            f"Exporting to ONNX opset version {opset_version} is not officially supported. "
```

Comment:
I thought it was resolved.

Reply:
We still don't have symbolic implementations for aten ops that use Reduce* operators. Setting opset>17 will cause models that use those ops to fail